packages/amplify_authenticator/lib/src/widgets/form_fields/confirm_sign_in_form_field.dart

```diff
@@ -90,26 +97,30 @@ abstract class ConfirmSignInFormField<FieldValue>
             hintText == null ? InputResolverKey.customAuthChallengeHint : null,
         field: ConfirmSignInField.customChallenge,
         validator: validator,
+        autofillHints: autofillHints,
       );
 
   /// Creates a verification code component.
   static ConfirmSignInFormField verificationCode({
     Key? key,
     FormFieldValidator<String>? validator,
+    Iterable<String>? overrideAutofillHints,
```

Comment:
I would rather name these all the same `autofillHints` since we may add defaults on other fields in the future and the other parameters to these functions already override defaults.

Reply:
Thanks Dillon, I'll apply the change to name all inputs `autofillHints` instead of `overrideAutofillHints`.